packages/components/pagination/src/page-size-selector/page-size-selector.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import PageSizeSelector, { type TPageRangeSize } from './page-size-selector';
+
+const getMinimumPageSizeFromRange = (selectedRange: TPageRangeSize) => {
+  switch (selectedRange) {
+    case 'l':
+      return 200;
+    default:
+      return 20;
+  }
+};
+
+const meta = {
```

Comment:
This story is not working correctly locally for me.
I'd like to have a sync about it. 🙏

Reply:
Ok, thank you, I'll schedule a meeting. I just wonder, it any different from https://uikit.commercetools.com/?path=/story/components-pagination--pagesizeselector?

Reply:
Something I ![image](https://github.com/commercetools/ui-kit/assets/97907068/777a7ef5-7676-4b8d-8895-b95452e234d9) is that using `inline: false` will make the controls to not work.
Perhaps we can wrap the story in a component which applies that height.

![image](https://github.com/commercetools/ui-kit/assets/97907068/2f14e663-716b-4ae6-b36d-765dc6893bf9)

Reply:
I found out that maybe using the `height` property can work:
![image](https://github.com/commercetools/ui-kit/assets/97907068/df312f02-9dd7-41fd-84e4-3299bc69451f)

Reply:
Thank you, I updated the story to use this parameter. However, I noticed a down side of using it. It works well in the `Docs` page and the story page, but it doesn't work in the "README" page. That is why I removed the  `<Canvas of={PaginationStories.Default} />` from the pagination's readme.mdx.

Reply:
According to [the docs](https://storybook.js.org/docs/api/doc-block-canvas#story), it seems we can adjust the height of the story the `Canvas` component renders.

Testing it locally, this example seems to work:
```jsx
<Canvas of={PaginationStories.Default} story={{ height: '560px' }}/>
```

![image](https://github.com/commercetools/ui-kit/assets/97907068/83fc5bff-d4aa-4816-9403-92de505b77f6)

The downside is that I had to use a high value for the height, but I think it's a fair compromise right now and maybe can take a deeper look in the future.

Reply:
Thanks. From my observations, the story within `<Canvas>` treats the viewport as its lower boundary. 
Whether using high or low values, the story may be perceived as not working as expected, depending on the current scroll position on the page.
![2024-01-24 09 44 01](https://github.com/commercetools/ui-kit/assets/49066275/6607bdb7-8c2d-4b2c-a7ba-0f5cc8e36d32)

Reply:
Maybe then we can just add a wrapper for the component in the story?